app/utils/discogs.ts

```diff
@@ -1,25 +1,33 @@
 import { SearchResponseSchema } from "./discogs.schemas";
 
-const discogs = {
-  search: async () => {
+type props = string;
+type searchResult = {
+  results: [{ year?: string }];
+};
+
+const token = process.env.DISCOGS_API_KEY!
+console.log("🚀 ~ file: discogs.ts:9 ~ token:", token)
+
 
-    // define url for GET API call
+async function discogs(search: props): Promise<searchResult>  {
```

Comment:
you can also just inline this type

```suggestion
async function discogs(search: string): Promise<searchResult>  {
```